FrameworksAndDrivers/Database/DotLanches.Data/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DotLanches.DataMongo.Repositories;
+using DotLanches.Domain.Interfaces.Repositories;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Bson.Serialization.Serializers;
+using MongoDB.Driver;
+
+namespace DotLanches.DataMongo.Extensions
+{
+    public static class ServiceCollectionExtensions
+    {
+        private const string DATABASE_NAME = "dotlanche";
```

Comment:
Esse nome precisa ser atualizado para ser exclusivo deste serviço
```suggestion
        private const string DATABASE_NAME = "dotlanche-pedido";
```